types/src/core/config.d.ts

```diff
@@ -27,5 +28,6 @@ export interface Config {
     ignoreNullOrigin?: boolean;
     nacl?: nacl;
     timelineParams?: any;
+    reportDeathCallback?: (info: Object) => void;
```

Comment:
@dkarella I'm wondering if a name like `failedConnEventCallback` and `failedConnEvent` would be more clear indicating failure of the ws and not the whole service? Just a thought...

Reply:
I think naming in this case isn't too important since it's a temporary change, but I do kind of prefer `reportDeathCallback` in this case since the function we're getting information from is called `reportDeath` and it's only triggered when the connection fails under certain conditions (not on all failures)

Reply:
@dkarella Makes sense!